docs/content/users/providers/lagoon.md

```diff
@@ -0,0 +1,18 @@
+# Amazee Lagoon Integration
```

Comment:
```suggestion
# Lagoon Integration
```

Lagoon exists as a stand-alone open source project and we want the integration to work for any Lagoon installation, so we can remove amazee.io as a call out.